vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/TextFieldIT.java

```diff
@@ -15,126 +15,35 @@
  */
 package com.vaadin.flow.component.textfield.tests;
 
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.demo.ComponentDemoTest;
 import org.junit.Assert;
 import org.junit.Test;
 import org.openqa.selenium.By;
 import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebElement;
 
-import com.vaadin.flow.component.radiobutton.testbench.RadioButtonGroupElement;
-import com.vaadin.flow.component.textfield.TextField;
-import com.vaadin.flow.component.textfield.testbench.TextFieldElement;
-import com.vaadin.flow.demo.ComponentDemoTest;
-
-import static com.vaadin.flow.data.value.ValueChangeMode.EAGER;
-
 /**
  * Integration tests for the {@link TextField}.
  */
 public class TextFieldIT extends ComponentDemoTest {
```

Comment:
For moving the tests, I think it would have been easiest and most clear to simply move the old demo pages to integration-tests and remove the dependency from integration-tests to the demo.

Now you have mostly moved the old demo parts to inside the other IT-views, and still depending on the demos on some views. This is really hard to follow.

We should remove all the dependencies from tests to demo views, either by
1. Keeping the old demo views as they are but just moving them inside integration-tests, or
2. Moving all the old demo parts to the existing integration tests views (like you have done for some parts but not all)